src/main/java/io/vertx/mqtt/MqttClientOptions.java

```diff
@@ -272,6 +273,17 @@ public MqttClientOptions setSsl(boolean ssl) {
     return this;
   }
 
+  @Override
+  public MqttClientOptions setTrustStoreOptions(JksOptions options) {
+    super.setTrustStoreOptions(options);
+    return this;
+  }
+
+  @Override
+  public MqttClientOptions setTrustAll(boolean trustAll) {
+    super.setTrustAll(trustAll);
+    return this;
+  }
```

Comment:
As already provided by the server component, even if without tests, we should add even the following ways to provide certs (as PEM, PFX, ...) overriding following methods : 

`setKeyCertOptions`
`setKeyStoreOptions`
`setPfxKeyCertOptions`
`setPemKeyCertOptions`
`setTrustOptions`
`setPemTrustOptions`
`setPfxTrustOptions`

and even adding following override to be more complete : 

`addEnabledCipherSuite`
`addEnabledSecureTransportProtocol`
`addCrlPath`
`addCrlValue`